lib/fluent/plugin/filter_throttle.rb

```diff
@@ -68,12 +83,26 @@ def configure(conf)
 
       raise "group_warning_delay_s must be >= 1" \
         unless @group_warning_delay_s >= 1
+
+      hostname = Socket.gethostname
+      expander_builder = Fluent::Plugin::Prometheus.placeholder_expander(log)
+      expander = expander_builder.build({ 'hostname' => hostname, 'worker_id' => fluentd_worker_id })
+      @base_labels = parse_labels_elements(conf)
+      @base_labels.each do |key, value|
+        unless value.is_a?(String)
+          raise Fluent::ConfigError, "record accessor syntax is not available in metric labels for throttle plugin"
+        end
+        @base_labels[key] = expander.expand(value)
+      end
+
     end
 
     def start
       super
 
       @counters = {}
+      # TODO add more relevant metrics to throttling
+      @metrics = {throttle_rate_limit_exceeded: get_counter(:fluentd_throttle_rate_limit_exceeded, "The exceeded rate of pods in the group")}
```

Comment:
this can be a variable. I dont think we need a map here

Reply:
This was made as a map just to make it standard to add more metrics. It can act as a template to add more metrics by adding in the map